app/controllers/recipie_controller.rb

```diff
@@ -1,9 +1,13 @@
 class RecipieController < ApplicationController
   before_action :authenticate_user!
   def show
-    @recipie = Recipie.find(params[:id])
+    @recipie = Recipie.find(params[:id])  
     @steps = @recipie.steps
+    @comments = Comment.all
+    @currentuser = current_user
+    @users = User.all
```

Comment:
Again fetching all users records here is not a good practice. Try to fetch only required records.

Reply:
Removed user.all and fetching only users commented . DONE